tests/Integrations/PCNTL/PCNTLTest.php

```diff
@@ -81,6 +82,25 @@ public function testCliShortRunningTracingWhenEnabled()
         ]);
     }
 
+    public function testAccessingTracerAfterForkIsUnproblematic()
+    {
+        list($output, $traces) = $this->inCliWithOutput(
+            __DIR__ . '/scripts/access-tracer-after-fork.php',
+            [
+                'DD_TRACE_CLI_ENABLED' => 'true',
+                'DD_TRACE_SHUTDOWN_TIMEOUT' => 5000,
```

Comment:
Is this shutdown necessary or it is just leftover?

Reply:
All the other tests are using it as well, hence I included it (blindly).

Reply:
Will take a note t look into it, maybe is not necessary anymore (I'd say hopefully) :)